packages/peregrine/lib/Apollo/policies/temporary.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Custom type policies that allow us to have more granular control
+ * over how ApolloClient reads from and writes to the cache.
+ *
+ * https://www.apollographql.com/docs/react/caching/cache-configuration/#typepolicy-fields
+ * https://www.apollographql.com/docs/react/caching/cache-field-behavior/
+ */
+const temporaryTypePolicies = {
+    StoreConfig: {
+        fields: {
+            // This field is available in Magento 2.4.2 we need to support older versions, so env var is used here
```

Comment:
Since we support `n` now, instead of `n-1`, do we need to mock this? By the time this code is released in PWA Studio, 2.4.2 should be released so I think this is fine to remove.